crane/container.go

```diff
@@ -262,6 +263,14 @@ func (r *RunParameters) Restart() string {
 	return os.ExpandEnv(r.RawRestart)
 }
 
+func (r *RunParameters) SecurityOpt() []string {
+	var flags []string
+	for _, rawFlag := range r.RawSecurityOpt {
+		flags = append(flags, os.ExpandEnv(rawFlag))
+	}
+	return flags
```

Comment:
I prefer specific names. Maybe we can find a generic solution for these options which are handled all the same, but so far I haven't come up with one for Go. Until then I think it's better to use e.g. `securityOpt` here.